src/portable/portablectl.c

```diff
@@ -769,13 +770,50 @@ static int maybe_stop_enable_restart(sd_bus *bus, sd_bus_message *reply) {
         return 0;
 }
 
-static int maybe_stop_disable(sd_bus *bus, char *image, char *argv[]) {
+static int maybe_clean_units(sd_bus *bus, char **units) {
+        int r;
+
+        assert(bus);
+        assert(units);
+
+        if (!arg_clean)
+                return 0;
+
+        STRV_FOREACH(name, units) {
+                _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+                _cleanup_(sd_bus_message_unrefp) sd_bus_message *m = NULL;
+
+                r = bus_message_new_method_call(bus, &m, bus_systemd_mgr, "CleanUnit");
+                if (r < 0)
+                        return bus_log_create_error(r);
+
+                r = sd_bus_message_append(m, "s", *name);
+                if (r < 0)
+                        return bus_log_create_error(r);
+
+                r = sd_bus_message_append_strv(m, STRV_MAKE("all"));
```

Comment:
Why not to all set `fdstore`?

Reply:
fixed typo in the previous comment.
`all` clears all directories, but not clears fdstore. So if you want to clear really _all_ info, you need to specify `all` and `fdstore`.

Reply:
oh I missed that, it's not good that 'all' doesn't mean 'all'... @poettering was that intentional when you added the fdstore clean?

Reply:
I guess `fdstore` was added after we released with `all`. To keep backward compat, fdstore is not included in all.

Reply:
added by hand for now

Reply:
> oh I missed that, it's not good that 'all' doesn't mean 'all'... @poettering was that intentional when you added the fdstore clean?

Nope, I think "all" shoudl cover fdstore too. Seems like oversight to me, if i didn't update that

Reply:
Yeah, I think `all` does cover fdstore?